geode-docs/tools_modules/redis_api_for_geode.html.md.erb

```diff
@@ -17,55 +17,54 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-The Redis API for <%=vars.product_name%> allows <%=vars.product_name%> to function as a drop-in replacement for a
+The <%=vars.product_name%> API for Compatibility with Redis allows <%=vars.product_name%> to function as a drop-in replacement for a
 highly-available Redis data store, letting Redis applications take advantage of
-<%=vars.product_name%>’s scaling capabilities without changing their client code. Redis clients connect to a <%=vars.product_name%>
+<%=vars.product_name%>’s scaling capabilities without changing their client code. Compatible with Redis Clients connect to a <%=vars.product_name%>
 server in the same way they connect to a Redis server, using a hostname and a port number, with
 optional password authentication.
 
 <img src="../images/redis_api_for_geode.png" class="image" />
 
-## <a id="using-the-api" class="no-quick-link"></a>Using the Redis API for <%=vars.product_name%>
+## <a id="using-the-api" class="no-quick-link"></a>Using the <%=vars.product_name%> API Compatible with Redis
 
-The <%=vars.product_name%> cluster must have at least one server that is set up to handle the incoming Redis
-commands.
+The <%=vars.product_name%> cluster must have at least one server that is set up to handle the incoming Compatible with Redis commands.
```

Comment:
Similar to the above, it might make sense to leave this as `Redis commands` since we are actually issuing Redis commands (there is no special set of `Compatible with Redis commands`.